MBBSEmu/HostProcess/GlobalRoutines/SysopGlobal.cs

```diff
@@ -198,5 +219,62 @@ private void RemoveKey(IReadOnlyList<string> commandSequence)
 
             _sessions[_channelNumber].SendToClient($"\r\n|RESET||WHITE||B|Removed Key {key} from User {userName}|RESET|\r\n".EncodeToANSIString());
         }
+
+        /// <summary>
+        ///     Sysop Command to broadcast a message to all users
+        ///
+        ///     Syntax: /SYSOP BROADCAST MESSAGE
+        /// </summary>
+        /// <param name="commandSequence"></param>
+        private void Broadcast(IReadOnlyList<string> commandSequence)
+        {
+            if (commandSequence.Count() < 3)
+            {
+                _sessions[_channelNumber].SendToClient("\r\n|RESET||WHITE||B|Invalid Command -- Syntax: /SYSOP BROADCAST <MESSAGE>|RESET|\r\n".EncodeToANSIString());
+                return;
+            }
+
+            var message = string.Join(" ", commandSequence.Skip(2));
+            
+            foreach (var c in _sessions)
+                if (c.Value.Channel != _channelNumber)
+                    _sessions[c.Value.Channel].SendToClient($"|RESET|\r\n|B||RED|SYSOP BROADCAST: {message}|RESET|\r\n".EncodeToANSIArray());
+        }
+
+        /// <summary>
+        ///     Sysop Command to kick the specified user
+        ///
+        ///     Syntax: /SYSOP KICK USER
+        /// </summary>
+        /// <param name="commandSequence"></param>
+        private void Kick(IReadOnlyList<string> commandSequence)
+        {
+            if (commandSequence.Count() < 3)
+            {
+                _sessions[_channelNumber].SendToClient("\r\n|RESET||WHITE||B|Invalid Command -- Syntax: /SYSOP KICK <USER>|RESET|\r\n".EncodeToANSIString());
+                return;
+            }
+
+            var userName = commandSequence[2];
+
+            if (_sessions.Values.Any(
+                s => string.Equals(s.Username, userName, StringComparison.CurrentCultureIgnoreCase)))
+            {
+                var channelToKick = _sessions.Values.FirstOrDefault(u =>
+                    u.Username.Equals(userName, StringComparison.InvariantCultureIgnoreCase));
+
+                //Not sure i need this second check -- IDE was worried?
+                if (channelToKick != null)
```

Comment:
`FirstOrDefault` will return null if no record is found.

You can invert this IF statement where if `channelToKick == null` display error and return. Would reduce nesting.